scripts/valhalla_build_config

```diff
@@ -551,18 +552,6 @@ help_text = {
   }
 }
 
-#used to convert comma separated list of strings into a list
-def comma_separated_list(arg):
```

Comment:
so we already suppored lists via comma separated strings, we just didnt support it for `optional`